src/app/modals/modals.component.ts

```diff
@@ -46,6 +46,7 @@ export class ModalsComponent implements DoCheck, OnInit {
   syncPercentage: number = 0;
   syncString: string;
   closeOnEscape: boolean = true;
+  disableClose: object; // @TODO CREATE model for object
```

Comment:
Yes name changes, and I had define Object in service so we can add more property later in the same object.